frontend/src/services/course-modules-api/course-modules-api.service.ts

```diff
@@ -0,0 +1,36 @@
+import { ApiPath, HttpMethod } from 'common/enums/enums';
+import {
+  CourseModuleGetByIdResponseDto,
+  CourseModuleGetRequestParamsDto,
+} from 'common/types/types';
+import { Http } from 'services/http/http.service';
+
+type Constructor = {
+  http: Http;
+  apiPrefix: string;
+};
+
+class CourseModules {
+  #http: Http;
+
+  #apiPrefix: string;
+
+  public constructor({ http, apiPrefix }: Constructor) {
+    this.#http = http;
+    this.#apiPrefix = apiPrefix;
+  }
+
+  public getById({
+    courseId,
+    moduleId,
+  }: CourseModuleGetRequestParamsDto): Promise<CourseModuleGetByIdResponseDto> {
+    return this.#http.load(
+      `${this.#apiPrefix}/${ApiPath.COURSES}/${courseId}/modules/${moduleId}`,
```

Comment:
After pulling changes from dev branch, please, replace /modules as text with CoursesApiPath.MODULES (I believe, it will be right move)